skycoin/skywire
# Update Angular to v14.2.11

Did you run `make format && make check`?
The go code was not changed. `npm run lint` and `npm run build` were used.

 Changes:	
- Angular was updated from v14.2.0 to v14.2.11
- A deprecation with the way in which scss files are accessed was fixed.

How to test this PR:
The UI must be rebuilt with `make build-ui`, but `make install-deps-ui` must be ran first. The UI should continue working as before.
